app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt

```diff
@@ -12,10 +15,16 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import com.google.android.material.snackbar.Snackbar
 import com.imaginaryrhombus.proctimer.R
+import com.imaginaryrhombus.proctimer.application.TimerEndNotification
+import com.imaginaryrhombus.proctimer.application.TimerNotification
 import com.imaginaryrhombus.proctimer.databinding.TimerFragmentBinding
 import com.imaginaryrhombus.proctimer.ui.timerpicker.TimerPickerFragment
 import kotlinx.android.synthetic.main.timer_fragment.*
 import org.koin.androidx.viewmodel.ext.android.sharedViewModel
+import java.text.SimpleDateFormat
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.util.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/b31a11cf7f617ce8ac48044c6c809fd38815309c/app/src/main/java/com/imaginaryrhombus/proctimer/ui/timer/TimerFragment.kt#L27"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>